framework/doc/sqa_reports.yml

```diff
@@ -38,5 +38,6 @@ Requirements:
     moose_test:
         directories:
             - test/tests
-        log_testable: WARNING
         show_warning: false
+        include_non_testable: true
+        log_testable: WARNING
```

Comment:
@bwspenc This will re-enable the checking for skips/deleted tests. Use "ERROR" if you don't want to be allowed, the SQA Check will then fail.